src/Utils.js

```diff
@@ -16,10 +17,72 @@ exports.humanHashrate = function(hashes) {
     return hashes.toFixed(1) + ' ' + units[u];
 }
 
+exports.stdConfig = {
+    address: 'NQ32 473Y R5T3 979R 325K S8UT 7E3A NRNS VBX2',
+    host: 'eu.sushipool.com',
+    port: '443',
+    name: os.hostname(),
+    consensus: 'dumb',
+    hashrate: 100,
+    devices: [0],
+    memory: [2048],
+    threads: [2],
+    cache: [3],
+}
+
+// https://regex101.com/r/hpK5w8/2
+exports.validateAddress = function(addr) {
+    const r = /(NQ[0-9]{2}\s(?:[a-zA-Z0-9]{4}\s?){8})/;
+    const res = r.exec(addr);
+    return res && res.length > 0;
+}
+
+exports.validateConfigFile = function(cfg) {
+    if (!exports.validateAddress(cfg.address)) {
+        throw new Error(`Could not validate address: '${cfg.address}'`);
+    }
+
+    if (['eu', 'us', 'asia'].indexOf(cfg.host.split('.')[0])) {
+        throw new Error(`Could not validate host: '${cfg.host}'`);
+    }
+
+    if (cfg.port !== '443') {
+        throw new Error(`Please use port '443' got '${cfg.port}'`);
```

Comment:
Port 443 is sushipool specific, better would be checking if it's an int.

Reply:
Fixed in `c81ae68749d66b1133c1631936883519af84840b`